cpuid/src/transformer/amd.rs

```diff
@@ -8,14 +8,61 @@ use kvm::CpuId;
 use kvm_bindings::KVM_CPUID_FLAG_SIGNIFCANT_INDEX;
 use transformer::common::use_host_cpuid_function;
 
+// Largest extended function.
+const LARGEST_EXTENDED_FN: u32 = 0x8000_001f;
+// This value allows 64 logical threads within a package.
+const THREAD_ID_MAX_SIZE: u32 = 6;
+// This value means there is 1 node per processor.
+const NODES_PER_PROCESSOR: u32 = 0;
```

Comment:
Is this supposed to be 0? If yes, could you please explain where exactly along the way this value becomes 1?

Reply:
Yes, it is supposed to be 0, The possible values are:
```
        // The number of nodes per processor. Possible values:
        // 0 -> 1 node per processor
        // 1 -> 2 nodes per processor
        // 2 -> Reserved
        // 3 -> 4 nodes per processor
```

I added this info as a comment for leaf_0x8000001e::ecx::NODES_PER_PROCESSOR_BITRANGE and mentioned this here.